test/e2e/flask/multichain-api/testHelpers.ts

```diff
@@ -0,0 +1,113 @@
+import * as path from 'path';
+import { DAPP_URL, largeDelayMs, openDapp, unlockWallet } from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+
+/**
+ * Default options for setting up Multichain E2E test environment
+ */
+export const DEFAULT_OPTIONS = {
+  dapp: true,
+  dappPaths: [
+    path.join(
+      '..',
+      '..',
+      'node_modules',
+      '@metamask',
+      'test-dapp-multichain',
+      'build',
+    ),
+  ],
+};
+
+/**
+ * Unlocks a wallet and provides extension id for dapp to connect to wallet extension.
+ *
+ * @param {Driver} driver - E2E test driver {@link Driver}, wrapping the Selenium WebDriver.
+ * @param extensionId - Extension identifier for web dapp to interact with wallet extension.
+ */
+export async function openDappAndConnectWallet(
+  driver: Driver,
+  extensionId: string,
+): Promise<void> {
+  await unlockWallet(driver);
+  await openDapp(driver, undefined, DAPP_URL);
+
+  await driver.fill('[placeholder="Enter extension ID"]', extensionId);
+  await driver.clickElement({ text: 'Connect', tag: 'button' });
+  await driver.delay(largeDelayMs);
+}
+
+/**
+ * Retrieves permitted session scopes by using test driver to interact with web dapp.
+ *
+ * @param {Driver} driver - E2E test driver {@link Driver}, wrapping the Selenium WebDriver.
+ * @returns result containing sessions scopes.
+ */
+export async function getSessionScopes(
+  driver: Driver,
+): Promise<{ sessionScopes: Record<string, unknown> }> {
+  await driver.clickElement({ text: 'wallet_getSession', tag: 'span' });
+
+  const completeResultSummary = await driver.findElements('.result-summary');
+
+  const getSessionResultSummary = completeResultSummary[0];
+  await getSessionResultSummary.click();
```

Comment:
this click may not be necessary

Reply:
It actually is, otherwise the element I want to fetch the result from will be empty (I was battling with this yesterday until I figured I had to click this)